examples/integrate.rs

```diff
@@ -33,6 +34,7 @@ struct State(SeatState<State>);
 impl SeatHandler for State {
     type KeyboardFocus = EguiState;
     type PointerFocus = EguiState;
+    type TouchFocus = WlSurface;
```

Comment:
Looking at the egui code, it seems they also support touch inputs now, so ideally we would want to update this integration and example to support it...

Well that can be done with another PR though, gonna make an issue for it.